### PR TITLE
Remove security alias for zh security.md

### LINKED ISSUE #18691: kubernetes.io/security forwards to the chinese security policy
**This is a Bug Report**

**Problem:**
kubernetes.io/security redirects to https://kubernetes.io/zh/docs/reference/issues-security/security/, but should redirect to https://kubernetes.io/docs/reference/issues-security/security/

### PR DESCRIPTION
**Pull request by gavinfish**
Try to fix #18691.

### CODE REVIEW

File: content/zh/docs/reference/issues-security/security.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
 ---
 title: Kubernetes 安全和信息披露
-aliases: [/security/]
```
Comment (tanjunchen) [inline content/zh/docs/reference/issues-security/security.md]: remove ?
it seems that we should sync en/ zh/.
image

image

could you sync them? Thanks

Comment (gavinfish) [reply content/zh/docs/reference/issues-security/security.md]: The aliases are used to create redirects. Ref https://gohugo.io/content-management/urls/#aliases .

**Pull request merged**